src/components/PupilComponents/QuizQuestionStem/QuizQuestionStem.tsx

```diff
@@ -4,25 +4,16 @@ import {
   OakSpan,
 } from "@oaknational/oak-components";
 
+import { QuizQuestionStemProps } from "../QuizResultQuestionStem/QuizResultQuestionStem";
```

Comment:
I 'm guessing sonar cloud made you do this, but it's not good. The props should not be shared between these two components.

Reply:
solution?

Reply:
I suggest it further down in the comments. Give me a call if unsure